package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "serverless-offline-sns",
-  "version": "0.52.0",
+  "version": "0.53.0",
```

Comment:
had to up this in order to install the package from my branch since the npm reg is on 0.53.0

Reply:
[npm link](https://docs.npmjs.com/cli/link) is your friend here! You can point to a local package, instead of bumping the version and resolving NPM locally